topcoffea/data/fromTTH/lepSF/README.md

```diff
@@ -0,0 +1,67 @@
+# topcoffea
```

Comment:
Looks like this one is a copy of the main README. The other for the b-tag UL SFs looks good, thanks!